### ISSUE TITLE
🚨 repo is older than version pushed to the playstore 🚨

### ISSUE THREAD

**Issue by Darkcast** [RfidResearchGroup/RFIDtools#90]
Hey @xianglin1998

I noticed that the version of this repo is 1.4.8 SDK21 && PM3 CWD when compilled it doesnt have the BT correct permissions for android OS 14 and the app crashes when giving it the permissions requested, The version in the play store 1.4.9 which has the permission works pretty well as long as you have the OLD client/fw for the proxmark3 X. 

I was wondering if you could upload the most recent version 1.4.9 so we can try to update the pm3 client and see if we can get it to run on the newer version.

Thanks for all the help

**Comment by Ten0** [comment]
This seems to work for me on that regard:
https://github.com/RfidResearchGroup/RFIDtools/pull/92

Although I've also had other issues that blocked the build, so really I'm building on https://github.com/Ten0/RFIDtools/tree/fix_build without flashing support.

Finally after ~5h of work I've managed to get this working so I'll save the struggle to anyone else who's trusting enough to run this:
https://mega.nz/file/nM1TzKRC#LRvR6M92RNcKSuW1kGdbPoKmp54BR0PErG9TdC5unrE

**Comment by whywilson** [comment]
> This seems to work for me on that regard: #92
> 
> Although I've also had other issues that blocked the build, so really I'm building on https://github.com/Ten0/RFIDtools/tree/fix_build without flashing support (so, flash via computer to current version - [ATM v4.16717](https://github.com/RfidResearchGroup/RFIDtools/commit/03dd487595efd58163c02830d0a821d5e7b43dbd)).
> 
> Finally after ~5h of work and hundreds of gigabytes of android tooling download I've managed to get this working so I'll save the struggle to anyone else who's trusting enough to use this `apk`: https://mega.nz/file/vBNyGZxb#umfWvKM2YChf8sRZS7t8zEZBYerLZE09euvNpihkWCc

Thank you for the hard working. What's the client version in App and what fw version should I use on Proxmark3?

**Comment by Ten0** [comment]
4.16.717

That being said quickly after someone pointed out that there's an alternate setup that works directly with termux :
https://github.com/RfidResearchGroup/proxmark3/blob/master/doc/termux_notes.md

That's a more up to date client.

**Comment by siowu** [comment]
> 在这方面对我来说这似乎有用： #92
> 
> 虽然我也遇到了其他阻碍构建的问题，所以实际上我是在https://github.com/Ten0/RFIDtools/tree/fix_build上构建的，没有闪存支持（因此，通过计算机闪存到当前版本 - [ATM v4.16717](https://github.com/RfidResearchGroup/RFIDtools/commit/03dd487595efd58163c02830d0a821d5e7b43dbd)）。
> 
> 最后，经过约 5 小时的工作和数百 GB 的 android 工具下载后，我终于成功完成了这个工作，所以我将把这个难题留给任何足够信任使用它的人`apk`： https://mega.nz/file/vBNyGZxb#umfWvKM2YChf8sRZS7t8zEZBYerLZE09euvNpihkWCc

Hello, this compiled version does not seem to support use after Bluetooth connection, have you tried it?

**Comment by Ten0** [comment]
No I haven't, I've only tried with proxmark3 connected via USB.

**Comment by siowu** [comment]
> 没有，我只尝试过通过 USB 连接 proxmark3。

Sorry, it's my problem. While using Bluetooth connection, I also connected the device to the computer, which caused the port to be occupied, so I can't use Bluetooth connection. In addition, I also want to know how to compile the client's SO file in order to enable Python scripts support on the client. Recently, the Proxmark3 project has launched a new Python script to crack RF08S, and I want to run it on an Android phone using RFIDTools😊

**Comment by Ten0** [comment]
> Recently, the Proxmark3 project has launched a new Python script to crack RF08S, and I want to run it on an Android phone using RFIDTools

https://github.com/RfidResearchGroup/RFIDtools/issues/90#issuecomment-2277218526

**Comment by siowu** [comment]
Yes, I attempted to compile the client via Termux; however, without root privileges, connecting to proxmark3 via Termux necessitates the use of third-party TCP/UART software, which is quite inconvenient.

> > Recently, the Proxmark3 project has launched a new Python script to crack RF08S, and I want to run it on an Android phone using RFIDTools
> 
> [#90 (comment)](https://github.com/RfidResearchGroup/RFIDtools/issues/90#issuecomment-2277218526)

**Comment by Crypt0Fox** [comment]
Wow.. Thank God I found this post 😂🤣😅
But mostly - thank you guys! ❤️

**Comment by H3d45** [comment]
> > > This seems to work for me on that regard: [#92](https://github.com/RfidResearchGroup/RFIDtools/pull/92)
> > > Although I've also had other issues that blocked the build, so really I'm building on https://github.com/Ten0/RFIDtools/tree/fix_build without flashing support (so, flash via computer to current version - [ATM v4.16717](https://github.com/RfidResearchGroup/RFIDtools/commit/03dd487595efd58163c02830d0a821d5e7b43dbd)).
> > > Finally after ~5h of work and hundreds of gigabytes of android tooling download I've managed to get this working so I'll save the struggle to anyone else who's trusting enough to use this `apk`: https://mega.nz/file/vBNyGZxb#umfWvKM2YChf8sRZS7t8zEZBYerLZE09euvNpihkWCc
> > 
> > 
> > The cloud storage service is no longer working.
> 
> Damn. I'll try helping you out in the near future... Gotta find it In my archives (in which though...) might take time, if you see I'm not back here - please do remind 🙏🏼 I also might get too busy and forget...

Did you find it, my friend?

**Comment by Crypt0Fox** [comment]
> > > > This seems to work for me on that regard: [#92](https://github.com/RfidResearchGroup/RFIDtools/pull/92)
> > > > Although I've also had other issues that blocked the build, so really I'm building on https://github.com/Ten0/RFIDtools/tree/fix_build without flashing support (so, flash via computer to current version - [ATM v4.16717](https://github.com/RfidResearchGroup/RFIDtools/commit/03dd487595efd58163c02830d0a821d5e7b43dbd)).
> > > > Finally after ~5h of work and hundreds of gigabytes of android tooling download I've managed to get this working so I'll save the struggle to anyone else who's trusting enough to use this `apk`: https://mega.nz/file/vBNyGZxb#umfWvKM2YChf8sRZS7t8zEZBYerLZE09euvNpihkWCc
> > > 
> > > 
> > > The cloud storage service is no longer working.
> > 
> > 
> > Damn. I'll try helping you out in the near future... Gotta find it In my archives (in which though...) might take time, if you see I'm not back here - please do remind 🙏🏼 I also might get too busy and forget...
> 
> Did you find it, my friend?

Hio, sorry for the delay, I believe one of these should work. don't remember which:

https://drive.google.com/file/d/16AQnFcD73ua5jGc31im14VV3agBD1PJ_/view?usp=sharing, https://drive.google.com/file/d/1jcPP4bqfAXw-gtjxxv5DYPuAWe6njm3o/view?usp=sharing
